lib/screens/login/login.dart

```diff
@@ -22,13 +27,21 @@ class LoginScreen extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    SharedPreferences sharedPreferences = sl.get<SharedPreferences>();
```

Comment:
since this variable is needed only when LoginSuccess, declare it where it is used.

Reply:
removes this for now

Reply:
Then when are you planning to do this?

Reply:
Done!